pkgs/applications/misc/sakura/default.nix

```diff
@@ -1,27 +1,37 @@
-{ stdenv, fetchurl, cmake, pkgconfig, gtk3, perl, vte, pcre2, glib , makeWrapper }:
+{ stdenv
+, fetchurl
+, cmake
+, pkgconfig
+, gtk3
+, perl
+, vte
+, pcre2
+, glib 
+, makeWrapper }:
```

Comment:
Please try avoid unnecessary formatting changes. It makes seeing the real changes harder and backports are not as easy.

Reply:
There's actually hide whitespace changes on github and with git also.

Reply:
Doesn't seem to work here.